Source/inv.cpp

```diff
@@ -358,9 +358,9 @@ void CheckInvPaste(int pnum, Point cursorPosition)
 	auto &player = Players[pnum];
 
 	SetICursor(player.HoldItem._iCurs + CURSOR_FIRSTITEM);
-	int i = cursorPosition.x + (IsHardwareCursor() ? 0 : (icursW / 2));
-	int j = cursorPosition.y + (IsHardwareCursor() ? 0 : (icursH / 2));
-	Size itemSize { icursW28, icursH28 };
+	int i = cursorPosition.x + (IsHardwareCursor() ? 0 : (icursSize.width / 2));
+	int j = cursorPosition.y + (IsHardwareCursor() ? 0 : (icursSize.height / 2));
+	Size itemSize { icursSize28 };

```

Comment:
I think we normally style it as:
```suggestion
	Size itemSize = icursSize28;
```

Reply:
I've read that this style is preferred with modern C++ and have been trying to incorporate it when I remember to.

Should I change it?

Reply:
probably not